pkgs/development/python-modules/joblib/default.nix

```diff
@@ -1,29 +1,35 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , sphinx
 , numpydoc
 , pytest
+, python-lz4
 }:
 
 
 buildPythonPackage rec {
   pname = "joblib";
-  version = "0.12.1";
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "68e6128e4734196616a39e2d48830ec7d61551c7f5748849e4c91478d2444524";
+  version = "0.12.2";
+
+  # get full repository inorder to run tests
+  src = fetchFromGitHub {
+    owner = "joblib";
+    repo = pname;
+    rev = version;
+    sha256 = "0qckqsqa5m2f85hdm0q18i3aajn56vx47dam8wdzpvncgw5g68lv";
   };
 
-  checkInputs = [ sphinx numpydoc pytest ];
+  checkInputs = [ sphinx numpydoc pytest python-lz4 ];
```

Comment:
I think we can include `python-lz4` in `propagatedBuildInputs`. It's their "faster alternative to zlib and gzip for compressed serialization."